services/docker-compose.tools.yml

```diff
@@ -2,14 +2,14 @@ version: '3.4'
 services:
   # Maintenance services
   #--------------------------------------------------------------------
-  # celery workers monitor
-  flower:
-    image: ondrejit/flower:latest
-    command: --broker=amqp://${RABBITMQ_USER}:${RABBITMQ_PASSWORD}@rabbit:5672
-    ports:
-      - 5555:5555
-    depends_on:
-      - rabbit
+  # By default disabled: celery workers monitor
+  #flower:
+  #  image: ondrejit/flower:latest
+  #  command: --broker=amqp://${RABBITMQ_USER}:${RABBITMQ_PASSWORD}@rabbit:5672
+  #  ports:
+  #    - 5555:5555
+  #  depends_on:
+  #    - rabbit
```

Comment:
hmm sure about this?

Reply:
we start to have far to many extra services that we are not using ... ok i will re-enable it